### PR TITLE
Update desktop download page for 22.10 release

### PR DESCRIPTION
**Pull request by carkod**
## Done

Updates for 22.10.

## QA

**Only 22.10 release updates**. I could not close the other comments, they were already addressed. Download links won't be ready until the release date

- Go to https://ubuntu-com-12174.demos.haus/download/desktop
- If demo doesn't build, please checkout this branch and `dotrun` it.
- Compare against [copy doc](https://docs.google.com/document/d/1ScuTtdrm3-iAvizTDxksVTAerV6QkCUN8Sg60uUlJsk/edit#)

## Issue / Card

Fixes https://github.com/canonical/ubuntu.com/issues/12161

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

### CODE REVIEW

File: templates/download/desktop/index.html [hunk 1]
```diff
@@ -55,6 +55,36 @@ <h2>Ubuntu {{ releases.lts.full_version }} LTS</h2>
...
 </section>
 
+<section class="p-strip--light">
+  <div class="row">
+
+    <div class="col-8 p-divider__block">
+      <h2>Ubuntu {{ releases.latest.full_version }}</h2>
+      <p class="u-no-padding--top">The latest version of the Ubuntu operating system for desktop PCs and laptops, Ubuntu 22.10 comes with nine months of security and maintenance updates, until July 2023.</p>
+      <p>Recommended system requirements are the same as for Ubuntu {{ releases.lts.short_version }}.</p>
+      <p><a href="{{ releases.latest.release_notes_url }}">Ubuntu {{ releases.lts.short_version }} LTS release notes</a></p>
```
Comment (sowasred2012) [inline templates/download/desktop/index.html:65]:
```suggestion
      <p><a href="{{ releases.latest.release_notes_url }}">Ubuntu {{ releases.latest.short_version }} release notes</a></p>
```

**Review by sowasred2012 — CHANGES_REQUESTED** [review]
There's a few places where LTS data is being used on the new release section, and we do need the new mascot image asset.

File: templates/download/desktop/index.html [hunk 2]
```diff
@@ -55,6 +55,36 @@ <h2>Ubuntu {{ releases.lts.full_version }} LTS</h2>
   </div>
 </section>
 
+<section class="p-strip--light">
+  <div class="row">
+
+    <div class="col-8 p-divider__block">
+      <h2>Ubuntu {{ releases.latest.full_version }}</h2>
+      <p class="u-no-padding--top">The latest version of the Ubuntu operating system for desktop PCs and laptops, Ubuntu 22.10 comes with nine months of security and maintenance updates, until July 2023.</p>
+      <p>Recommended system requirements are the same as for Ubuntu {{ releases.lts.short_version }}.</p>
```
[the remark this answers is not in the document]
Comment (carkod) [reply templates/download/desktop/index.html:64]: If you look at the copy docs, this sentence is talking about `Recommended system requirements are the same as for Ubuntu 22.04 LTS` so this is LTS not latest.

File: templates/download/desktop/index.html [hunk 3]
```diff
@@ -55,6 +55,36 @@ <h2>Ubuntu {{ releases.lts.full_version }} LTS</h2>
...
+          alt="",
+          width="260",
+          height="150",
+          hi_def=True,
+          loading="auto"
+          ) | safe
+        }}
+      </div>
+      <p>
+        <a class="p-button--positive is-wide" href="/download/desktop/thank-you?version={{ releases.lts.full_version }}&amp;architecture=amd64" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Download', 'eventAction' : 'Desktop', 'eventLabel' : '{{ releases.lts.short_version }}', 'eventValue' : undefined });">Download</a>
```
[the remark this answers is not in the document]
Comment (carkod) [reply templates/download/desktop/index.html:80]: This is all correct, we are still keeping the 22.04 on the page, no changes, I just moved the code. What I don't know is if the `22.04.1` is the LTS or the `22.04`

**Comment by sowasred2012** [comment]
@carkod here's the link for the asset: https://assets.ubuntu.com/v1/38ba52dc-Kinetic+Kudo+22.10.svg

**Comment by carkod** [comment]
Also Pdm asked for design https://docs.google.com/document/d/1VJ3xT2ViC-CvL0UmsAUzpYFmvpuqUP4AbpnlPm8tqOA/edit?disco=AAAAfH7iX6E

**Comment by carkod** [comment]
> @carkod here's the link for the asset: https://assets.ubuntu.com/v1/38ba52dc-Kinetic+Kudo+22.10.svg

All done. By the way, are you able to close the old comments/suggestions?

File: releases.yaml [hunk 4]
```diff
@@ -16,6 +16,16 @@ lts:
...
+previous_latest:
+  slug: ImpishIndri
+  name: "Impish Indri"
+  short_version: "21.10"
+  full_version: "21.10"
+  core_version: "22"
+  release_date: "October 2021"
+  eol: "July 2022"
+  past_eol_date: true
+  release_notes_url: "https://discourse.ubuntu.com/t/impish-indri-release-notes/21951"
```
Comment (sowasred2012) [inline releases.yaml:28]: I don't think we need this? It doesn't look like this data is consumed anywhere, it's only LTS's that have historical data.

**Review by sowasred2012 — CHANGES_REQUESTED** [review]
Looks good, but there's some data in there I don't think we need - also it looks like there's a bad merge, the PR has commits for /developers and /flavours

File: releases.yaml (hunk 4 above)
Comment (carkod) [reply releases.yaml]: Right. Removed.

**Comment by carkod** [comment]
> Looks good, but there's some data in there I don't think we need - also it looks like there's a bad merge, the PR has commits for /developers and /flavours

yeah, rebased.

File: templates/download/desktop/index.html [hunk 5]
```diff
@@ -55,6 +55,37 @@ <h2>Ubuntu {{ releases.lts.full_version }} LTS</h2>
   </div>
 </section>
 
+<section class="p-strip--light">
+  <div class="row">
+
+    <div class="col-8 p-divider__block">
+      <h2>Ubuntu {{ releases.latest.full_version }}</h2>
+      <p class="u-no-padding--top">The latest version of the Ubuntu operating system for desktop PCs and laptops, Ubuntu 22.10 comes with nine months of security and maintenance updates, until July 2023.</p>
```
Comment (sowasred2012) [inline templates/download/desktop/index.html:63]:
```suggestion
      <p class="u-no-padding--top">The latest version of the Ubuntu operating system for desktop PCs and laptops, Ubuntu {{ releases.latest.short_version }} comes with nine months of security and maintenance updates, until {{ releases.latest.eol }}.</p>
```

**Review by sowasred2012 — APPROVED** [review]
Looks good, just a couple of minor changes before it's merged

**Pull request merged**